recipes/r-rcppplanc/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# 'Autobrew' is being used by more and more packages these days
+# to grab static libraries from Homebrew bottles. These bottles
+# are fetched via Homebrew's --force-bottle option which grabs
+# a bottle for the build machine which may not be macOS 10.9.
+# Also, we want to use conda packages (and shared libraries) for
+# these 'system' dependencies. See:
+# https://github.com/jeroen/autobrew/issues/3
+export DISABLE_AUTOBREW=1
+
+#disable clang availability checks
+export CMAKE_ARGS="${CMAKE_ARGS} -DCMAKE_CXX_FLAGS=-D_LIBCPP_DISABLE_AVAILABILITY"
```

Comment:
Please use the invocation given in the link in the error message. i.e.
```
export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
```

Reply:
https://github.com/conda-forge/staged-recipes/commit/9ad5d667dbea3c2cb2fba46b68ce58906ac23386

I tried. My configure script didn't like it.

Reply:
Why these lines https://github.com/welch-lab/RcppPlanc/blob/master/configure#L37-L38  ?

Reply:
To keep the build flags limited to the ones present in R's configuration.